packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.tsx

```diff
@@ -224,6 +228,7 @@ export class DetailsRow extends BaseComponent<IDetailsRowProps, IDetailsRowState
         style={ { minWidth: viewport ? viewport.width : 0 } }
         aria-selected={ isSelected }
         allowFocusRoot={ true }
+        isInnerZoneKeystroke={ (ev: React.KeyboardEvent<HTMLElement>) => (this.props.innerZoneFocusible ? ev.which === getRTLSafeKeyCode(KeyCodes.down) : false) }
```

Comment:
Why not have the user pass in the `isInnerZoneKeystroke` callback instead of only allowing it to go to `down`?

Reply:
On setting ```isInnerZoneKeystroke``` to ```right ```, navigation inside row was breaking.
When focus is on row, on pressing right focus comes to an item inside row.
Pressing right again, it will enter sub-item. Focus will not come out of sub-items on further right press.
The siblings of item inside the row wont be keyboard accessible.

Reply:
I think you should instead use the `Enter` key for this. Using `Down` is not very intuitive.

Reply:
```Down``` key has been recommended by the Homeland Security Certified Testers.
It is also being used in VSTS.